### DISCUSSION TITLE
Looking for an AS3 lib for XMP Metadata access (parsing and optionally modifying)

### DISCUSSION THREAD

**Discussion by hnuecke** [airsdk/Adobe-Runtime-Support#2698] [Q&A]
I know there was a lib available at "Adobe times" (as3_xmp_file and XMPCore), which is no longer available.
The only SDKs still supported are for other languages than AS3 (Java, C#).
I could not fid any question/answer here that is related to Metadata handling (or I did not search correctly).

BTW, I know how to parse standard EXIF Data in JPG Files. But before going down the XMP path also, I thought I'd better ask if not somebody else solved that before ;-)

**Comment by marchbold** [comment]
Haven't tried this one myself, but there appears to be the old xmp lib in the google code archive:

https://code.google.com/archive/p/as3-xmp-file/

Might at least be a starting point?
